LayoutTests/http/wpt/pointerlock/pointerlock_unadjustedMovement.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<style type="text/css">
+    button {
+        color: blue;
+    }
+
+    #target-wrap {
+        position: relative;
+        background-color: lightgrey;
+        width: 200px;
+        height: 100px;
+        border: grey 1px solid;
+    }
+
+    #target {
+        position: relative;
+        background-color: lightyellow;
+        width: 100px;
+        height: 30px;
+        border: yellow 1px solid;
+    }
+
+    #status-log {
+        margin: 10px 0;
+        color: green;
+    }
+</style>
+</head>
+<body>
+    <h2>Description</h2>
+    <p>This test validates that pointer lock accepts the unadjustedMovement option.</p>
+    <hr/>
+
+    <h2>Manual Test Steps:</h2>
+    <p>
+        <ol>
+            <li>Click the "lockTarget" button to request a pointer lock.</li>
+            <li>Test is done.</li>
+        </ol>
+    </p>
+    <hr/>
+
+    <button id="Button">lockTarget</button>
+
+    <div id="target-wrap">
+        <div id="status-log">Click the "lockTarget" button.</div>
+        <div id="target">Target</div>
+    </div>
+    <hr/>
+
+    <div id="log"></div>
+
+    <script type="text/javascript" >
+        let button = document.getElementById('Button');
+        let target = document.getElementById('target');
+
+        async_test(t => {
+            button.addEventListener('mousedown', t.step_func(async () => {
```

Comment:
This seems to be testing multiple independent things in a single step... should this be split?

The test should be fine to fail the feature is unsupported.

Reply:
Got it. I will let the test expectation be that it fails without unadjustedMovement support (so will be expected to fail on GTK for instance).